client/test/syntaxes/basic_predicate.sw.snap

```diff
@@ -0,0 +1,22 @@
+>predicate;
+#^^^^^^^^^ source.sway source.sway meta.attribute.sway
+#         ^ source.sway
+>
+>fn main() -> bool {
+#^^ source.sway meta.function.definition.sway keyword.other.fn.sway
+#  ^ source.sway meta.function.definition.sway
+#   ^^^^ source.sway meta.function.definition.sway entity.name.function.sway
+#       ^ source.sway meta.function.definition.sway punctuation.brackets.round.sway
+#        ^ source.sway meta.function.definition.sway punctuation.brackets.round.sway
+#         ^ source.sway meta.function.definition.sway
+#          ^^ source.sway meta.function.definition.sway keyword.operator.arrow.skinny.sway
+#            ^ source.sway meta.function.definition.sway
+#             ^^^^ source.sway meta.function.definition.sway entity.name.type.primitive.sway
+#                 ^ source.sway meta.function.definition.sway
+#                  ^ source.sway meta.function.definition.sway punctuation.brackets.curly.sway
+>    true
+#^^^^ source.sway
+#    ^^^^ source.sway constant.language.bool.sway
+>}
+#^ source.sway punctuation.brackets.curly.sway
+>
```

Comment:
GitHub will complain about missing new lines at the end of files

Reply:
These are generated files, so we shouldn't modify them by hand. I think they're ignored by the linter since the file extension is `.snap`

Reply:
ah, makes sense! in that case, disregard my previous comments haha